### ISSUE TITLE
Link new taxon name to ZooBank Registration

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#2150]
Just saw at talk at mammal meetings about ZooBank registration for new taxon names. Also just received a publication citing Arctos specimens as new species, and new taxon has a ZooBank registration. I've added this to new taxon source authority. Should we come up with a way to link directly and/or explicitly? They are primarily focused on newly described species, but may be adding legacy taxa. Discussion of taxon concepts was central here.

http://www.zoobank.org/

http://authorservices.taylorandfrancis.com/wp-content/uploads/2017/07/Registering-new-animal-names-in-ZooBank.pdf

urn:lsid:zoobank.org:act:15566E70-
921B-473E-840E-11D796F2E3C4

**Comment by campmlc** [comment]
Test taxon
https://arctos.database.museum/name/Linognathoides%20urocitelli

**Comment by dustymc** [comment]
Media can link most anything to actually anything.

I can't make that LSID resolve, so I don't see anything to link to.

**Comment by campmlc** [comment]
Suggestions to give to the ZooBank folks that might make it possible?

**Comment by dustymc** [comment]
DOIs would be my choice, but I don't think this is strictly a technology issue.  https://github.com/ArctosDB/arctos/issues/2144

Apparently http://zoobank.org/urn:lsid:zoobank.org:act:15566E70-921B-473E-840E-11D796F2E3C4 is supposed to work, but I can't find anywhere that's documented and it 404s for me.

**Comment by campmlc** [comment]
The error says that ZooBank was recently moved to a new server. I emailed
Richard Pyle at the Bishop Museum to ask what the next step would be.

**Comment by campmlc** [comment]
Message and Response from Richard Pyle:

Hi Richard,
This is Mariel from the GenBank to Redlist symposium at ASM. I enjoyed your talk, and I'm sorry I didn't get a chance to catch up with you afterwards. 
I'd like to ask you about the possibility of linking ZooBank registration IDs to  Arctos taxonomic names. After I got back from the conference, we received a paper with museum citations describing a new parasite species that has a ZooBank registration. We created the new name in our taxonomic structure, but when we try to create the ZooBank link given in the publication as a url, we get a 404 error saying the database has moved to a new server.. Do you have a new link we can use? I'd very much like to be able to link the Arctos taxonomic names to ZooBank identifiers whenever possible.

Here is the ID from the manuscript:
urn:lsid:zoobank.org:act:15566E70-921B-473E-840E-11D796F2E3C4 

and the link we have created: 
http://zoobank.org/urn:lsid:zoobank.org:act:15566E70-921B-473E-840E-11D796F2E3C4 
  

Thanks!
Mariel


--

**Comment by campmlc** [comment]
Hi Mariel, 

Many thanks, and I likewise enjoyed your talk! It’s funny, because I were also thinking the same thing (finding a way to link Arctos records to ZooBank).  We can think about this more generally, but in the specific case you mention, the notice about the server move is actually unrelated.  That’s the live server, but I had that notice up for a while in case there were any weird issues.  It’s been stable for weeks now, so I took down the notice.

But the missing record is something else –the ZooBank record hasn’t been updated to reflect the fact that it was published.  This is not uncommon – all “In Press” registrations are hidden from public view.  After they’re published, the author or journal editor is supposed to update the ZooBank record to reflect this, but many times they fail to do this, or delay in doing it, so records remain hidden on the website.  In other words, the link is correct, but the record in ZooBank needs to be updated to reflect the fact that it has been published.

I assume this is the correct publication:
https://bioone.org/journals/Journal-of-Parasitology/volume-105/issue-3/18-198/A-New-Species-of-Sucking-Louse-from-the-Long-Tailed/10.1645/18-198.short
(The title was different in the ZooBank record.)

I’ve now updated the ZooBank record to show that it was published on June 28, so it is now publicly available (and the link should work). 

Incidentally, because the  biodiversity informatics community never really adopted LSIDs, we’ll probably be phasing them out in favor of the root UUIDs, so the best link to use would take the form of:

http://zoobank.org/15566E70-921B-473E-840E-11D796F2E3C4

We’re committed to continuing to resolve all flavors of the link, but the plain on (without the LSID prefix) is the preferred link

Please let me know if any of that makes sense…

Aloha,

Rich

 

Richard L. Pyle, PhD
Database Coordinator | Associate Zoologist | Dive Safety Officer

Bernice Pauahi Bishop Museum

1525 Bernice Street, Honolulu, HI 96817-2704

Office: (808) 848-4115;  Fax: (808) 847-8252

eMail: deepreef@bishopmuseum.org

BishopMuseum.org

**Comment by campmlc** [comment]
@dustymc

**Comment by dustymc** [comment]
That's surprisingly enlightening! (People are the problem - whoodathunkit...)

So there's something to link, that's what Media does, here's the mini-tutorial.

<img width="125" alt="Screen Shot 2019-07-09 at 6 51 14 PM" src="https://user-images.githubusercontent.com/5720791/60934694-1ac0d400-a27c-11e9-82f7-0e82ed48cb56.png">
<img width="159" alt="Screen Shot 2019-07-09 at 6 52 03 PM" src="https://user-images.githubusercontent.com/5720791/60934702-1f858800-a27c-11e9-8b85-c4a967cf838d.png">
<img width="529" alt="Screen Shot 2019-07-09 at 6 52 53 PM" src="https://user-images.githubusercontent.com/5720791/60934704-214f4b80-a27c-11e9-8da7-1e8970139d97.png">
<img width="200" alt="Screen Shot 2019-07-09 at 6 52 59 PM" src="https://user-images.githubusercontent.com/5720791/60934708-23b1a580-a27c-11e9-87a2-6b425c94e8fa.png">
<img width="92" alt="Screen Shot 2019-07-09 at 6 53 16 PM" src="https://user-images.githubusercontent.com/5720791/60934712-26ac9600-a27c-11e9-884a-45a8775ce0ac.png">
<img width="225" alt="Screen Shot 2019-07-09 at 6 53 25 PM" src="https://user-images.githubusercontent.com/5720791/60934714-28765980-a27c-11e9-9c15-065e3aef8536.png">
<img width="648" alt="Screen Shot 2019-07-09 at 6 56 28 PM" src="https://user-images.githubusercontent.com/5720791/60934717-2a401d00-a27c-11e9-9af2-b09ae6fe1bde.png">
<img width="1123" alt="Screen Shot 2019-07-09 at 6 57 03 PM" src="https://user-images.githubusercontent.com/5720791/60934722-2ca27700-a27c-11e9-9b42-3a94e5eb5ea5.png">

**Comment by Jegelewicz** [comment]
Just came back to this while prepping for Taxonomy Committee. It is really COOL and I wish I had run across this three months back, before adding over 300 classifications to Arctos and putting the url in the "source authority" field instead of adding as media.

BUT it seems like A LOT of work when we could just be using a base url like we do in the Other IDs.

I just completed this process for [Myliobatidae](https://arctos.database.museum/name/Myliobatidae) and it turns out kinda nice - I added the IRMNG logo as the preview so that people can see what they are getting and also added IRMNG to agents so we can see how many taxa in Arctos are sourced from IRMNG. This creates a lot of interesting info but also means that there are now two steps in creating any classification that is sourced from IRMNG, GBIF, ZooBank, etc.

If we had a TAXON_TERM called "IRMNG Taxon Number" with the base url "http://www.irmng.org/aphia.php?p=taxdetails&id=" then we put "114692" in the value, we could create the link to IRMNG. We could do the same with "ZooBank Taxon Number".

**Comment by Jegelewicz** [comment]
Either way - This needs documentation and distribution to the community.

**Comment by dustymc** [comment]
>A LOT of work 

That's almost always a UI problem. Eg I think you can drag-n-drop an image then click a button to create taxonomy-media (and a bunch of other kinds), there's no reason that form can't also accept a URL.

Media centralizes monitoring for link-rot and all that jazz in one place designed to handle one datatype. Scattering that out would be a LOT more work to maintain, and we'd end up with some combination of a million different taxon terms, mixed data (which is essentially impossible to maintain), some new complex code table that'll eventually end up as an infinite-ish list of taxonomy-things, ....

tl;dr: I think Media is the correct tool for this job, and we can always build better UIs.
